packages/tooltip/src/Tooltip.js

```diff
@@ -7,8 +7,9 @@ import alignments from "./constants/alignments";
 
 export const getAutoAlign = bodyWidth => containerRef => {
   if (containerRef) {
-    const { x } = containerRef.getBoundingClientRect();
-    if (x > bodyWidth / 2) return alignments.RIGHT;
+    const { x } = containerRef.getBoundingClientRect(); // Position of the element on viewport
+    const halfViewport = bodyWidth / 2;
+    if (x > halfViewport) return alignments.RIGHT;
```

Comment:
Ok... So the logic here is: if the element is on the "second" half of the screen, it's gonna be right aligned no matter what:
![image](https://user-images.githubusercontent.com/17936244/38087021-f589ab86-332c-11e8-947d-65ccd1460d4c.png)

I first thought that this feature was supposed to calculate if the tooltip content fits on the screen. If it doesn't fit the screen, then we would right align it.

If this is actually the expected behaviour, please let it documented.